src/core/sdl/sdl_core.cpp

```diff
@@ -26,10 +26,13 @@
 
 bool CoreSDL::init(const CmdLineSetup_t &setup)
 {
+    (void)(setup);
+
     bool res;
 
-    if(setup.allowBgInput)
-        SDL_SetHint(SDL_HINT_JOYSTICK_ALLOW_BACKGROUND_EVENTS, "1");
+    // if(g_config.background_work)
+    // apply this unconditionally -- otherwise, need to restart game for background-work to function as expected
+    SDL_SetHint(SDL_HINT_JOYSTICK_ALLOW_BACKGROUND_EVENTS, "1");
```

Comment:
This thing is an option, because I bet, there are some people who don't want it being backgroud input at all. For example, some other software can also use gamepad too and then cause the game to be poked unintentionally. Anyway, this thing needs some more info to make a proper choice on this.